docs/gcloud-auth.rst

```diff
@@ -43,3 +45,43 @@ instructions in the :ref:`Overview`. The credentials are inferred from your
 local environment by using Google `Application Default Credentials`_.
 
 .. _Application Default Credentials: https://developers.google.com/identity/protocols/application-default-credentials
+
+Credential Discovery Precedence
+-------------------------------
+
+When loading the `Application Default Credentials`_, the library will check
+properties of your local environment in the following order
+
+#. Application running in Google App Engine
+#. JSON or PKCS12/P12 keyfile pointed to by
+   ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable
+#. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
+#. Application running in Google Compute Engine
```

Comment:
Yup. Already in my notes to have a section which does the explicit version of all the ADC (app. default creds) stuff.